index.js

```diff
@@ -288,19 +216,29 @@ export default class OneSignal {
     }
 
     static getTags(next) {
-        if (!checkIfInitialized()) return;
+        if (!checkIfInitialized(RNOneSignal)) return;
 
         RNOneSignal.getTags(next);
     }
 
     static deleteTag(key) {
-        if (!checkIfInitialized()) return;
+        if (!checkIfInitialized(RNOneSignal)) return;
+        RNOneSignal.sendTag(key, "");
```

Comment:
Should keep `RNOneSignal.deleteTag(key);` as it accounts for this implementation detail

Reply:
what implementation detail?

Reply:
switched to use native deleteTag method anyway